server/chat/background/__init__.py

```
@@ -1,8 +1,8 @@
 """Background chat execution module for automated/webhook-triggered chats."""
 
 from .background_websocket import BackgroundWebSocket
-from .task import run_background_chat, cancel_rca_for_incident
+from .task import run_background_chat
 from .summarization import generate_incident_summary
 
-__all__ = ["BackgroundWebSocket", "run_background_chat", "cancel_rca_for_incident", "generate_incident_summary"]
+__all__ = ["BackgroundWebSocket", "run_background_chat", "generate_incident_summary"]
```

Ruff (0.16.2)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)